RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -198,5 +197,19 @@ private void configureButtonBindings() {
     buttons.outtake.whenPressed(intakeSubsystem::spinBackward);
     buttons.intake.whenReleased(intakeSubsystem::stopSpin);
     buttons.outtake.whenReleased(intakeSubsystem::stopSpin);
+
+    //Shooter BUttons
+    buttons.hubShoot.whenPressed(shooterSubsystem::shootTop);
+    buttons.hubShoot.whenReleased(shooterSubsystem::stopShootTop);
+
+    buttons.lowShoot.whenPressed(shooterSubsystem::shootLow);
+    buttons.lowShoot.whenReleased(shooterSubsystem::stopShootLow);
```

Comment:
I don't think we need separate stop* functions. Stop should just be "turn off the motors"

Reply:
done